app/models/meursing_lookup/steps/base.rb

```diff
@@ -0,0 +1,66 @@
+module MeursingLookup
+  module Steps
+    class Base < WizardSteps::Step
+      def self.inherited(child_class)
+        child_class.attribute child_class.key, :string
+        child_class.validates child_class.key, presence: true
+
+        super
+      end
+
+      def options
+        @options ||= begin
+          current_tree = tree
+
+          previous_answers.each.with_index do |answer, index|
+            if index >= previous_answers.size - 1
```

Comment:
This should be assigned to a variable name explaining what `index >= previous_answers - 1` is testing for - this would help explain what the conditional is for.